header.php

```diff
@@ -1,3 +1,16 @@
+<?php 
+
+
+    $user = wp_get_current_user()->data;
+    $meta = get_user_meta($user->ID);
+
+    if(isset($meta['wp_auth0_obj']) && sizeof($meta['wp_auth0_obj']) === 1) {
```

Comment:
This was removed in a previous PR, we could remove it here or remove it after the merge